### DISCUSSION TITLE
How to translate alias git='LC_ALL=en_GB git' to nushell?

### DISCUSSION THREAD

**Discussion by expensne** [nushell/nushell#11437] [Q&A] [answered]
As mentioned in the title, I wonder how to translate:
`alias git='LC_ALL=en_GB git'`
to nushell syntax.

Trying:
`alias git = LC_ALL=en_GB git`
results in _executable was not found_.

I am using nushell 0.88.1.

**Comment by amtoine** [comment, marked as answer]
you should be able to
- `alias git = with-env {LC_ALL: en_GB} { git }` => that'll likely lose autocompletions
- define `LC_ALL` in `env.nu` => that might not be desirable and might only apply to `git`
- define a custom command, something like
```nushell
def --wrapped git [...args] {
    $env.LC_ALL = "en_GB"
    ^git $args
}
```

> **Note**
> i don't really use `def --wrapped` but it looks like it might be working

**Reply by expensne** [reply]
Thank you for the quick reply.

> - alias git = with-env {LC_ALL: en_GB} { git } => that'll likely lose autocompletions

Does not allow arguments to be used, e.g. `git status` results in: _Extra positional argument._

> - define a custom command, something like
`
def --wrapped git [...args] {
    $env.LC_ALL = "en_GB"
    ^git $args
}
`

Works perfectly - thanks!
